commands/namespace_create.go

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) OpenFaaS Author(s) 2023. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/openfaas/faas-cli/util"
+	"github.com/openfaas/faas-provider/types"
+	"github.com/spf13/cobra"
+)
+
+// NamespaceCreateFlags holds flags that are to be added to commands.
+type NamespaceCreateFlags struct {
+	labelOpts      []string
+	annotationOpts []string
+}
+
+var namespaceCreateFlags NamespaceCreateFlags
+
+var namespaceCreateCmd = &cobra.Command{
+	Use: `create NAMESPACE_NAME
+			[--label LABEL=VALUE ...]
+			[--annotation ANNOTATION=VALUE ...]`,
+	Short: "Create a new namespace",
+	Long:  "Create command creates a new namespace",
+	Example: `faas-cli namespace create namespace-name
+	faas-cli namespace create namespace-name --label demo=true
+	faas-cli namespace create namespace-name --annotation demo=true
+	faas-cli namespace create namespace-name --label demo=true --annotation demo=true --gateway=http://127.0.0.1:8080`,
```

Comment:
Generally I like to wrap after 80 chars, so:

```
faas-cli namespace create namespace-name --label demo=true \
  --annotation demo=true \
  --gateway=http://127.0.0.1:8080
```

Although I'd leave off any examples with overridden gateways for brevity.

Reply:
Done